### DISCUSSION TITLE
Login directly with Proton (SIWSL)

### DISCUSSION THREAD

**Discussion by ph00lt0** [simple-login/app#1900] [Feature Requests]
I use the sign in with simplelogin SSO option.
I would like the option to configure my SIWSL app to always redirect to use "sign in with proton".
Currently my users first have to press sign in with simplelogin, followed by sign in with proton as they are all in a proton aossicated account. Would be awesome if you could remove this uncessary and confusing step for the end user.

**Comment by ghost** [comment]
In this case user will leak real email, not alias. The main goal of SL is to hide email under alias

**Reply by ph00lt0** [reply]
No you misunderstand the issue.
When you end up at the login screen of SL you need to press sign in with proton to log in to SL. 

It simply has nothing to do with what aliases is being used. Besides it being quite irrelevant to the discussion as this is about a self-controlled environment.
